src/annotator/range-util.js

```diff
@@ -129,3 +129,35 @@ export function selectionFocusRect(selection) {
     return textBoxes[textBoxes.length - 1];
   }
 }
+
+/**
+ * Retrieve a set of items associated with nodes in a given range.
```

Comment:
Sorry, I've lost the plot here a bit. What is an "item?" Perhaps this summary doc string could be expanded a little?

Reply:
"Item" can be any data that the caller which to associate with / compute from a node, as determined by the callback.